ALIGN-analoglayout/ALIGN-public
# Stubs for adding new placement injection function

@854768750 Here is a PR with the stub added.
Please work on the method: `void PnRdatabase::setPlacementInfoFromJson(const string& json_string)`

To test, you can try:
```
export ALIGN_HOME=/path/to/align-public
export ALIGN_WORK_DIR=$ALIGN_HOME/work
mkdir -p $ALIGN_WORK_DIR
cd $ALIGN_WORK_DIR
mkdir five_transistor_ota
cd five_transistor_ota
schematic2layout.py ../../examples/five_transistor_ota --reference_placement_verilog_json `pwd`/../../examples/five_transistor_ota/FIVE_TRANSISTOR_OTA_0.scaled_placement_verilog.json
```
Here, instead of running the placer Python procedure we are running this new C++ method with the example placement verilog json file. 
Let me know if you want this to be different. We can scale the JSON file differently if you want as well. Currently it should be in the units described by scale_factor in the PDK Json file.

It currently crashes because we are not populating the data structures yet.
